apps/user-office-frontend/cypress.json

```diff
@@ -2,7 +2,8 @@
   "video": false,
```

Comment:
Probably we don't need this cypress.json file. I think we use the one that is in the apps/user-office-frontend-e2e/ and this one is leftover after moving to the monorepo.

Reply:
done